docs/Community/governance/sig-charter-template.md

```diff
@@ -0,0 +1,82 @@
+SIG YOURSIG Charter
+===================
+
+## Scope
+
+Include a two to three sentence summary of the work this SIG will do. Imagine
+trying to explain it to someone who knows what Open Enclave SDK does but isn't
+yet familiar with all the internals.
+
+## In Scope
+
+### Code, Binaries, or Services
+
+- list of what qualifies as a deliverable (code, binary, or service)
+- that is within the scope of this SIG
+- for example, driver code for specific hardware,
+- or documentation .md files,
+- but DO NOT list specific code paths or locations here
+
+### Cross-cutting or Externally Facing Processes
+
+- list of processes owned by this SIG, such as
+- the creation of a periodic release .tar.gz file,
+- or the ongoing operation of CI/CD infrastructure,
+- or the management of a mentorship program.
+
+### Out of Scope
+
+List some things that might seem like they fall in-scope for this SIG, but don't (or don't yet).
+
+## Process & Management
+
+THIS SIG meets at least THIS OFTEN.
```

Comment:
SIG-Release does not meet at all. Are SIGs required to meet?

Reply:
I think we previously said no, SIGs are not required to ever meet

Reply:
I believe I'd said (and certainly intended to say) that SIGs do not have to meet. I can see how that's not clear in the charter-template.

That being the case, I also believe that there is value in having a transparent process, so a SIG which does not meet should endeavour to have a clearly communicated process via some other means. 

I'll think about the best way to indicate this flexibility in the charter....